bin/export_atlas_uniprot.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# What this script does:
+# 1. Gets all unique UniProt IDs in Atlas bulk-analytics collection
+# 2. Generates an Atlas search linnk for each UniProt ID
+# 3. Writes everything to file
+
+# Caveats:
+# 1. Only works for bulk-analytics for now. There seems to be no  in scxa solr (???) still waiting for reply from devs
```

Comment:
there is no scxa in dev solr.. we lost it hen we lost solr completely, but for testing try using public/fallback or dev on solr8